### ISSUE TITLE
Heroku deployment broken

### ISSUE THREAD

**Issue by joshghent** [emedvedev/slackin-extended#12]
Hi there, thanks for this great project and forking the original!

When clicking to deploy it to Heroku, you get the following error
```
The content of app.json is not valid; please see app.json Schema for more information.
```

**Comment by emedvedev** [comment]
Hey! Thanks a lot for the report, I'll look into it when I have the time, but it might take a while. In the meantime, did you do any debugging or looking into the issue? Any additional information (or a PR) would be super helpful. :)

**Comment by joshghent** [comment]
@emedvedev No debugging no, just when you click the deploy to heroku button in the read me, it fails. Sorry I should have been more clear. I'll look into creating a PR for fixing this if I get time 👍

**Comment by emedvedev** [comment]
Fixed by #16.

**Comment by jimprince** [comment]
I thought this was resolved but I'm getting a deployment error now: 
https://github.com/emedvedev/slackin-extended/pull/16#issuecomment-523287384
